src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -207,32 +207,73 @@ module Types = struct
               in
               Block_time.to_string @@ C.end_time ~constants global_slot ) ] )
 
+  let consensus_time_with_global_slot_since_genesis =
+    obj "ConsensusTimeGlobalSlot"
+      ~doc:"Consensus time and the corresponding global slot since genesis"
+      ~fields:(fun _ ->
+        [ field "consensusTime" ~typ:(non_null consensus_time)
+            ~args:Arg.[]
```

Comment:
Can you add doc string here that defines the format of this field? i.e. is it an integer that represents seconds since the UTC epoch?